src/client/components/ConsentsNavigation.tsx

```diff
@@ -43,7 +47,7 @@ export const ConsentsNavigation = ({
               {
                 page: previousPage,
               },
-              queryParams,
+              queryParamsWithCmpState,
```

Comment:
Since this variable is only used once, perhaps the spread operator could be done in line here.